skiko/src/jvmMain/resources/not-supported-adapter.list

```diff
@@ -0,0 +1,9 @@
+directx:Intel(R) HD Graphics 520;
```

Comment:
Please add human readable reason as last field, and show it when adapter is disabled in error message.